doc/instruction_decode_execute.rst

```diff
@@ -116,18 +116,18 @@ The Multiplier/Divider (MULT/DIV) is a state machine driven block to perform mul
 The fast and slow versions differ in multiplier only. All versions implement the same form of long division algorithm. The ALU block is used by the long division algorithm in all versions.
 
 Multiplier
-  The multiplier can be implemented in three variants controlled via the parameter ``MultiplierImplementation``.
+  The multiplier can be implemented in three variants controlled via the enumerated parameter ``RV32M`` defined in :file:`rtl/ibex_pkg.sv`.
 
   Single-Cycle Multiplier
-    This implementation is chosen by setting the ``MultiplierImplementation`` parameter to "single-cycle". The single-cycle multiplier makes use of three parallel multiplier units, designed to be mapped to hardware multiplier primitives on FPGAs. It is therefore the **first choice for FPGA synthesis**.
+    This implementation is chosen by setting the ``RV32M`` parameter to "RV32MSingleCycle". The single-cycle multiplier makes use of three parallel multiplier units, designed to be mapped to hardware multiplier primitives on FPGAs. It is therefore the **first choice for FPGA synthesis**.
```

Comment:
Please use the fully-qualified enum name.

While you're at it: can you make it one sentence per line?

Reply:
Done. Good catch about one sentence per line.